tests/behat/features/Admin/Events/GestionEvenements.feature

```diff
@@ -11,7 +11,7 @@ Feature: Administration - Évènements - Gestions Évènements
   @clearAllSponsorFiles
   Scenario: On crée un nouvel évènement vide
     Given I am logged in as admin and on the Administration
-    And I follow "Gestion évènements"
+    And I follow "afup-main-menu-item--forum_gestion"
```

Comment:
ce n'est pas dommage d'utiliser un élément technique alors qu'avant c'était plus fonctionnel ? Si le souci est un souci d'unicité du lien ne faudrait-il pas créer un "And I follow {link} in the admin menu" ?

Reply:
Le problème d'utiliser le label c'est que ça force à avoir chaque menu unique dans toute l'arborescence.

Par exemple dans la partie `Évènements` y'a `Factures d'évènement` et dans la partie `Trésorerie` y'a `Factures.`

Si je voulais avoir `Factures` dans les deux ce serait pas possible vu qu'on utilise le label pour cibler.

Ou alors on fait une phrase comme ça : `And I follow {link} in section {section} of the admin menu` ? Je peux tenter une PR pour ça, tu en penses quoi ?

Reply:
oui, la formulation `And I follow {link} in section {section} of the admin menu` me semble pas très bien (à voir si on utilise seulement celle-ci ou aussi in `And I follow {link} in the admin menu ?` pour les cas où on a pas de doublon, je n'ai pas d'avis tranché).